53-aditofsdk.rules

```diff
@@ -0,0 +1,12 @@
+# allow read/write access to the eeprom
```

Comment:
I think we can move this file in another place, maybe utils or misc, it shouldn't be in the top level directory

Reply:
moved to utils